sonar-plugin-api/src/main/java/org/sonar/api/batch/CpdMapping.java

```diff
@@ -19,20 +19,21 @@
  */
 package org.sonar.api.batch;
 
+import java.io.File;
+import java.util.List;
 import net.sourceforge.pmd.cpd.Tokenizer;
 import org.sonar.api.ExtensionPoint;
 import org.sonar.api.resources.Language;
 import org.sonar.api.resources.Resource;
 
-import java.io.File;
-import java.util.List;
-
 /**
  * Implement this extension to get Copy/Paste detection for your language.
  * @since 1.10
+ * @deprecated since 5.5
  */
 @BatchSide
 @ExtensionPoint
+@Deprecated
 public interface CpdMapping {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)